conf/test.config

```diff
@@ -37,7 +37,6 @@ params {
     input          = 'https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/testdata/samplesheet_trio.csv'
 
     // Genome references
-    fasta                = "https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/reference.fasta"
```

Comment:
you should still have fasta defined here

Reply:
Wouldn't that mess with config priority? 🤔

Reply:
Because parameters defined in main.nf has the lowest priority, if the parameter is not set anywhere else, I would want it to grab the file from igenomes. But if its defined in test.config, that gets higher priority and igenomes doesn't matter?